src/ClipboardManager.vala

```diff
@@ -244,13 +244,15 @@ namespace Marlin {
 
             switch (target_info) {
                 case ClipboardTarget.GNOME_COPIED_FILES: /* Pasting into a file handler */
+warning ("gnome copied");
```

Comment:
Where these intended to be removed after testing? If not maybe should be a `debug` since it's a little noisy

Reply:
Sorry, yes should have been removed.